go/saltpack/specs/saltpack_encryption.md

```diff
@@ -224,7 +242,7 @@ recipients from each other.
 
 We compute the authenticators in three steps:
 
-1. Concatenate the **header hash**, the nonce from the **payload secretbox**,
+1. Concatenate the **header hash**, the nonce for the **payload secretbox**,
```

Comment:
No, I haven't gone into it. The short version is the "associated data" in AEAD, yeah?